Source/WebCore/css/StyleProperties.cpp

```diff
@@ -1096,13 +1045,12 @@ String StyleProperties::borderImagePropertyValue(const StylePropertyShorthand& s
     StringBuilder result;
     bool omittedSlice = false;
     bool omittedWidth = false;
-    String commonWideValueText;
     auto separator = "";
     for (auto longhand : shorthand) {
         // All longhands should be present.
         auto value = getPropertyCSSValue(longhand);
         if (!value)
-            return String();
+            return { };
```

Comment:
This file has so many inconsistencies between `String()`, `nullString()`, `{ }`, and some places even using `emptyString()`. Choosing one and switching all may be better left for another patch?

Reply:
Sure I will revert this. Recently @anttijk told me we normally use this syntax and I think it stuck in my head.

The null vs. empty string is a more serious topic though, not just syntax.